src/app/views/apply/Apply.js

```diff
@@ -18,4 +18,4 @@ function Apply() {
     );
 }
 
-export default Apply;
\ No newline at end of file
+export { Apply as default, APPLICATION_URL };
```

Comment:
We can leave the default export and instead say `export const APPLICATION_URL = ...`